src/Kdyby/Doctrine/QueryBuilder.php

```diff
@@ -107,15 +107,27 @@ public function autoJoinOrderBy($sort, $order = NULL)
 		}
 
 		if (is_string($sort)) {
-			$alias = $this->autoJoin($sort);
-			$sort = $alias . '.' . $sort;
+			$reg = "#([a-zA-Z_. ]+)\)#";
```

Comment:
I think you should also add the `\(` to the beggining so the regexp at https://github.com/Kdyby/Doctrine/pull/206/files#diff-f6cc3ca901f9632af758a516a922ba63R117 can be nicer